pkg/freeze/containerd/pause_test.go

```diff
@@ -0,0 +1,144 @@
+package containerd
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/containerd/containerd"
+	"google.golang.org/grpc"
+	cri "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	"knative.dev/container-freezer/pkg/daemon"
+)
+
+const (
+	freezeMethod = "freeze"
```

Comment:
I don't know how much this const is really adding since it's only defined in the test, just makes me have to scroll further in the file